packages/http/src/forwarder/__tests__/index.spec.ts

```diff
@@ -4,11 +4,12 @@ import { assertResolvesRight, assertResolvesLeft } from '@stoplight/prism-core/s
 import { keyBy, mapValues } from 'lodash';
 import { hopByHopHeaders } from '../resources';
 import { DiagnosticSeverity, Dictionary } from '@stoplight/types';
+import { format } from 'url';
```

Comment:
i'm not seeing any functional changes here

Reply:
It is my mistake :)